src/components/UI/organisms/LoginUser.js

```diff
@@ -1,16 +1,19 @@
 import {Form, Button, Row, Col} from 'react-bootstrap';
 import React, { useState } from 'react';
 
-const LoginUser = ({onSubmit}) => {
+const LoginUser = (props) => {
```

Comment:
small nit but why did you replace with props? using `{onSubmit}` allows you to reference onSubmit directly later instead of having to use `props.`